src/Acquisition/tests.py

```diff
@@ -3278,7 +3314,7 @@ class B(Implicit):
         base = B()
         base.value = 42
 
-        rich_cmp_methods = ['__lt__', '__gt__', '__eq__',
+        rich_cmp_methods = ['__lt__', '__gt__',  # '__eq__',
```

Comment:
A textual comment would be helpful for a future reader why `__eq__` is omitted here.

Reply:
I reverted to the old behavior: ignore the base object's equality (and ordering) definitions (as this is implemented by the "C" version, too) -- see #50. Thus, this special handling of `__eq__` is gone.